### PR TITLE
Added option for 3rd dose

### PR DESCRIPTION
**Pull request by siddnikh**
Had to comment out a few lines in PatientRegister.tsx to allow commit. 
Solves #2037

### CODE REVIEW

**Comment by nihal467** [comment]
wpokdqowd

@gigincg @siddnikh it is not accepting 3rd dose

**Comment by gigincg** [comment]
@siddnikh Have you created a backend issue for this?

**Comment by siddnikh** [comment]
@nihal467 I had to update the backend too. Here's the PR for that: https://github.com/coronasafe/care/pull/775
Please try using that.

**Comment by siddnikh** [comment]
@gigincg yeah. You can check it out here: https://github.com/coronasafe/care/pull/775

**Comment by nihal467** [comment]
test approved

File: src/Components/Patient/PatientRegister.tsx [hunk 1]
```diff
@@ -25,11 +25,11 @@ import {
   GENDER_TYPES,
   MEDICAL_HISTORY_CHOICES,
   TEST_TYPE,
-  DESIGNATION_HEALTH_CARE_WORKER,
+  //DESIGNATION_HEALTH_CARE_WORKER,
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 2]
```diff
@@ -25,11 +25,11 @@ import {
...
   VACCINES,
 } from "../../Common/constants";
 import countryList from "../../Common/static/countries.json";
-import statesList from "../../Common/static/states.json";
+//import statesList from "../../Common/static/states.json";
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 3]
```diff
@@ -66,9 +66,9 @@ const PageTitle = loadable(() => import("../Common/PageTitle"));
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const debounce = require("lodash.debounce");
 
-const placesList = countryList.concat(
-  statesList.filter((i: string) => i !== "Kerala")
-);
+// const placesList = countryList.concat(
+//   statesList.filter((i: string) => i !== "Kerala")
+// );
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove unused variables and do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 4]
```diff
@@ -103,7 +103,7 @@ const diseaseStatus = [...DISEASE_STATUS];
 const bloodGroups = [...BLOOD_GROUPS];
 
 const testType = [...TEST_TYPE];
-const designationOfHealthWorkers = [...DESIGNATION_HEALTH_CARE_WORKER];
+//const designationOfHealthWorkers = [...DESIGNATION_HEALTH_CARE_WORKER];
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 5]
```diff
@@ -855,12 +855,12 @@ export const PatientRegister = (props: PatientRegisterProps) => {
...
-    const { checked, name } = e.target;
-    form[name] = checked;
-    dispatch({ type: "set_form", form });
-  };
+  // const handleCheckboxFieldChange = (e: any) => {
+  //   const form = { ...state.form };
+  //   const { checked, name } = e.target;
+  //   form[name] = checked;
+  //   dispatch({ type: "set_form", form });
+  // };
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

**Review by khavinshankar — CHANGES_REQUESTED** [review]
please do work on the requested changes,

**Note:**
- _the unused variables, functions, interfaces and imports can be removed instead of commenting them out_
- _do prioritize on fixing an eslint issue instead of using eslint disable_

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**